.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,20 @@
+name: Chromatic
+
+on: push
+
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
always worth commenting on why this is needed (for instance, `chromaui/action` needs it, also our `release` and `commit-watch` CLIs need it too)